packages/target-tools/src/telemetryProvider.js

```diff
@@ -24,31 +21,26 @@ export function TelemetryProvider(
 ) {
   let telemetryEntries = [];
 
-  function getMode(status) {
```

Comment:
We should actually keep this fn and modify it a bit.  Based on yesterday's discussion, this should do the trick:
```
if (status === OK && (decisioningMethod === DECISIONING_METHOD.ON_DEVICE || method === DECISIONING_METHOD.HYBRID)) {
      return EXECUTION_MODE.LOCAL;
} else {
     return EXECUTION_MODE.EDGE;
}
```